pkg/config/agentfeaturesconfig.go

```diff
@@ -24,6 +25,7 @@ type AgentFeaturesConfiguration struct {
 	PersistCache            bool              `config:"persistCache"`
 	MarketplaceProvisioning bool              `config:"marketplaceProvisioning"`
 	ExternalIDPConfig       ExternalIDPConfig `config:"idp"`
+	SdkAgentStatusUpdates   bool              `config:"sdkAgentStatusUpdates"`
```

Comment:
Can this be changed to enableAgentStatusUpdates?

Reply:
Can we rename it as agentStatusUpdates  without the "enable" to be consistent with all the other flags like (versionChecker, marketplaceProvisioning, connectToCentral , ...)?

Reply:
I agree that we should drop enable from the name.